docs/algorithms/sidb_simulation.rst

```diff
@@ -119,10 +119,13 @@ Energy Calculation
 
 .. tabs::
     .. tab:: C++
-        **Header:** ``fiction/algorithms/simulation/sidb/energy_distribution.hpp``
+        **Header:** ``fiction/algorithms/simulation/sidb/calculate_energy_distribution.hpp``
```

Comment:
Not entirely sure here because I'm on my phone right now. Isn't the header's name `calculate_energy_and_state_type.hpp`?

If I'm mistaken and `energy_distribution` and the `calculate_` function are in their own header, I would call that one `energy_distrinution` to be consistent with other headers like `operational_domain.hpp` for instance.

Reply:
good point! Let's call the header ``energy_distribution.hpp``